gke-deploy/core/gcs/gcs.go

```diff
@@ -28,17 +27,14 @@ type GCS struct {
 	GcsService services.GcsService
 }
 
-// Download copies file(s) from GCS. <dst> should be a directory, not a path to a file
+// Download copies file(s) from GCS. <dst> should be a directory, not a path to a file.
 func (s *GCS) Download(ctx context.Context, src, dst string, recursive bool) error {
-	log.Printf("Downloading file(s) %s from %s.\n", src, dst)
```

Comment:
Why delete the log?

Reply:
I'm trying to keep this file more consistent with the rest of the code, where we don't print too much of what's happening under the hood unless -V is passed.